src/index.js

```diff
@@ -0,0 +1,116 @@
+import './style.css';
+
+const list = document.querySelector('.list');
+const inputTodo = document.querySelector('.add-todo');
+const addTodoBtn = document.querySelector('.submit');
+
+class TodoTask {
+  constructor(desp, arr, done = false) {
+    this.desp = desp;
+    this.index = arr.length;
+    this.done = done;
+  }
+}
+
+class TaskLisk {
+  constructor() {
+    this.tasks = [];
```

Comment:
- [ ] Hi @samiullahbahadur, I can see that you are doing so great in this project :+1:  but from the requirements of this particular project of todo list app, tasks arrays should contain some simple to do tasks (array of objects) containing three keys: description, completed, and index. Please let's fix this.